### PR TITLE
Update 4.4.0 changelog with ruleset changes

### PR DESCRIPTION
**Pull request by 72nomada**
Updated 4.4 changelog with ruleset changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -215,11 +215,40 @@ All notable changes to this project will be documented in this file.
...
+
+- Adding support for new sysmon events. ([#13594](https://github.com/wazuh/wazuh/pull/13594))
+- Adding new detection rules using Sysmon ID 1 events. ([#13595](https://github.com/wazuh/wazuh/pull/13595))
+- Adding new detection rules using Sysmon ID 3 events. ([#13596](https://github.com/wazuh/wazuh/pull/13596))
+- Adding new detection rules using Sysmon ID 7 events. ([#13630](https://github.com/wazuh/wazuh/pull/13630))
+- Adding new detection rules using Sysmon ID 8 events. ([#13637](https://github.com/wazuh/wazuh/pull/13637))
+- Adding new detection rules using Sysmon ID 10 events. ([#13639](https://github.com/wazuh/wazuh/pull/13639))
+- Adding new detection rules using Sysmon ID 11 events. ([#13631](https://github.com/wazuh/wazuh/pull/13631))
+- Adding new detection rules using Sysmon ID 13 events. ([#13636](https://github.com/wazuh/wazuh/pull/13636))
+- Adding rules for Sysmon ID 20 events. ([#13673](https://github.com/wazuh/wazuh/pull/13673))
```
Comment (davidjiglesias) [inline CHANGELOG.md:230]: Why is this line using a different format with respect to all previous ones?

File: CHANGELOG.md [hunk 2]
```diff
@@ -215,11 +215,40 @@ All notable changes to this project will be documented in this file.
...
+- Adding new PowerShell ScriptBlock detection rules. ([#13638](https://github.com/wazuh/wazuh/pull/13638))
+- Added HPUX 11i SCA policies using bastille and without bastille. ([#15157](https://github.com/wazuh/wazuh/pull/15157))
```
Comment (davidjiglesias) [inline CHANGELOG.md:232]: What does it mean with and without bastille? It sounds weird.

File: CHANGELOG.md [hunk 3]
```diff
@@ -215,11 +215,40 @@ All notable changes to this project will be documented in this file.
...
+
+#### Changed
+
+- Update ruleset according to new API log changes when user logged in with authorization context. ([#15072](https://github.com/wazuh/wazuh/pull/15072))
```
Comment (davidjiglesias) [inline CHANGELOG.md:236]:
```suggestion
- Updated ruleset according to new API log changes when the user is logged in with authorization context. ([#15072](https://github.com/wazuh/wazuh/pull/15072))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -215,11 +215,40 @@ All notable changes to this project will be documented in this file.
...
+- Update 0580-win-security_rules.xml rules. ([#13579](https://github.com/wazuh/wazuh/pull/13579))
+- Update Wazuh MITRE ATT&CK database to version 11.3. ([#13622](https://github.com/wazuh/wazuh/pull/13622))
+- Updating detection rules in 0840-win_event_channel.xml. ([#13633](https://github.com/wazuh/wazuh/pull/13633))
```
Comment (davidjiglesias) [inline CHANGELOG.md:239]: It sounds a bit strange to use update or updating interchangeably.

**Review by davidjiglesias — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (72nomada) [reply CHANGELOG.md:230]: it is not a different format, it is a new file that wasn't present before. It is not detection related change.

File: CHANGELOG.md (hunk 2 above)
Comment (72nomada) [reply CHANGELOG.md:232]: Bastille is a HP-UX tool hardening tool. We have implemented both policies as there are HP-UX admins that doesn't use bastille at all.

File: CHANGELOG.md (hunk 1 above)
Comment (72nomada) [reply CHANGELOG.md]: Anyway, I did change it

File: CHANGELOG.md (hunk 4 above)
Comment (72nomada) [reply CHANGELOG.md]: changed

**Review by davidjiglesias — APPROVED** [review]

**Pull request merged**